zookeeper-server/src/main/java/org/apache/zookeeper/ZKWatchManager.java

```diff
@@ -52,7 +54,7 @@ class ZKWatchManager implements ClientWatchManager {
         this.defaultWatcher = defaultWatcher;
     }
 
-    void setDefaultWatcher(Watcher defaultWatcher) {
+    public void setDefaultWatcher(Watcher defaultWatcher) {
```

Comment:
Seems for test only. I suggest we move `ConfigWatcherPathTest` to `package org.apache.zookeeper;` and avoid this change.